src/components/PopupWithForm.js

```diff
@@ -1,17 +1,27 @@
 import Popup from "./Popup.js";
-
 export default class PopupWithForm extends Popup {
-  constructor(popupSelector, handleFormSubmit) {
+  constructor(popupSelector, handleFormSubmit, cardFormSubmit) {
     super({ popupSelector });
     this._popupForm = this._popupElement.querySelector(".modal__form");
     this._inputList = this._popupForm.querySelectorAll(".modal__input");
     this._handleFormSubmit = handleFormSubmit;
+    this._cardFormSubmit= cardFormSubmit;
```

Comment:
`  this._cardFormSubmit= cardFormSubmit;` is not used, and it not needed in the universal `PopupWithForm`